README.md

```diff
@@ -284,7 +284,30 @@ To request automatic tracing support for a module not on this list, please [file
 
 ## Upgrade guidelines
 
-### 0.24.x to x
+### 0.25.x to 0.26.x
+
+Collector exporter packages and types are renamed:
+
+- `@opentelemetry/exporter-collector` => `@opentelemetry/exporter-otlp-http`
+  - `CollectorExporterBase` => `OTLPExporterBase`
+  - `CollectorTraceExporter` => `OTLPTraceExporter`
+  - `CollectorMetricExporter` => `OTLPMetricExporter`
+  - `CollectorExporterBrowserBase` => `OTLPExporterBrowserBase`
+  - `CollectorExporterNodeBase` => `OTLPExporterNodeBase`
+  - `OTLPExporterConfigBase` => `OTLPExporterConfigBase`
+  - `OTLPExporterError` => `OTLPExporterError`
+  - `OTLP_SPAN_KIND_MAPPING` => `OTLP_SPAN_KIND_MAPPING`
+  - `otlpTypes` => `otlpTypes`
+- `@opentelemetry/exporter-collector-grpc` => `@opentelemetry/exporter-otlp-grpc`
+  - `OTLPTraceExporter` => `OTLPTraceExporter`
+  - `OTLPMetricExporter` => `OTLPMetricExporter`
+  - `OTLPExporterConfigNode` => `OTLPExporterConfigNode`
+- `@opentelemetry/exporter-collector-proto` => `@opentelemetry/exporter-otlp-proto`
+  - `OTLPExporterNodeBase` => `OTLPExporterNodeBase`
+  - `OTLPMetricExporter` => `OTLPMetricExporter`
+  - `OTLPTraceExporter` => `OTLPTraceExporter`
```

Comment:
The classes on the lefthand side should should be changed to their original names (with collector instead of OTLP)

Reply:
ah oops